src/stores/game.js

```diff
@@ -272,6 +272,8 @@ export const useGameStore = defineStore('game', {
     },
     successfullyJoined(player) {
       this.players.push(cloneDeep(player));
+      // sort players so p0 is at index 0 and p1 is at index 1 (bug repair)
```

Comment:
Let's put the comment above the push and explain both lines together
```suggestion
      // Add new player to game and sort based on pNum
      this.players.push(cloneDeep(player));
```